doc/Articles/FAQ.md

```diff
@@ -115,11 +115,73 @@ Using a list server means players do not have to manually enter IP addresses and
 You can show as much, or as little data as you like to the players, such as host name, type (deathmatch), player count (45 / 50), ping, enemy difficulty, map, region etc
 
 - Mirror Listserver: Mirror squad manage and host this for you, you pay monthly subscription, but do not have to worry about setting up or maintaining it.
-- Node ListServer: Free, but you host the files yourself on an unblocked PC, like a VPS. Has a wide variety of customisable features, best option if you are using dedicated hosted games.
+- Node ListServer: Free, but you host the files yourself on an unblocked PC, like a VPS. Has a wide variety of customisable features, best option if you want self hosted dedicated games.
 - Dark Reflective Mirror:  This is a list server and a relay, it is free, but you host and manage the files yourself.
 Relays offer an unblocked route of traffic, but at the cost of extra latency/increased ping. This relay will test for a direct connection first, then fallback to routed traffic if that fails. Both the list server and relayed traffic is optional, you can use one part, and not the other.
 This is the best option for player hosted games, where router port blocks are common.
     
 Simple Matchmaker
 - You can make matchmakers out of these list servers, simply hide the list to players, have them auto join a game with space. You could also filter out various requirements the player has set, for example, "USA" Region only, or "Lava Island" map.
     </details>
+    
+### How to get player count?
+
+<details>
+  <summary>There are a few ways to do this, each with their own unique benefit.</summary>
+    
+NetworkServer.connections.Count
+- Socket connections, includes people without spawned prefabs, non authenticated, or that may have bugged out but during spawn, but have connected temporarily (Android users minimising game).  Only host / server can check this.
+
+NetworkManager.singleton.numPlayers
+- Number of active spawned player objects across all connections on the server.
+Only host / server can check this, an example of how to sync this number to all players:
+```cs
```

Comment:
imho everything above this line is perfectly enough for a FAQ.
the code below this line is too much.
also tbh I didn't even know that NetworkManager can have a NetworkBehaviour field. it's definitely not best practice and not guaranteed to work forever 👍 
good FAQ. keep it simple. let's have the part above this line and merge it :)

Reply:
Ah its all separate scripts, i'l make it more clearer in the FAQ

Reply:
@JesusLuvsYooh please remove everything below this line then we can merge :)